mailbox/store/src/main/java/org/apache/james/mailbox/store/StoreMailboxManager.java

```diff
@@ -411,7 +411,7 @@ private boolean userHasLookupRightsOn(Mailbox mailbox, MailboxSession session) t
     }
 
     private void assertMailboxPathBelongToUser(MailboxSession mailboxSession, MailboxPath mailboxPath) throws MailboxException {
-        if (!mailboxPath.belongsTo(mailboxSession)) {
+        if (mailboxSession.getType() != MailboxSession.SessionType.System && !mailboxPath.belongsTo(mailboxSession)) {
```

Comment:
assertMailboxPathBelongToUser is now wrong.

`assertSessionHasAdministrativeRightsOn(MailboxSession mailboxSession, MailboxPath mailboxPath)` ?